presentation/build.gradle

```diff
@@ -75,22 +71,23 @@ android {
         jvmTarget = "1.8"
     }
 
-
     productFlavors {
         withAnalytics { dimension "analytics" }
         noAnalytics {
-            signingConfig signingConfigs.release
+//            signingConfig signingConfigs.release
         }
     }
+
     lint {
         abortOnError false
     }
+
     namespace 'dev.octoshrimpy.quik'
```

Comment:
Should we change this also?

Reply:
If I change the namespace it breaks A LOT of things and I talked with @unplugged-maksim and we decided to not rename the whole project structure and folders, so we will leave this in the code everywhere it's not user facing